kernel-install/50-mkosi-initrd.install

```diff
@@ -22,13 +22,19 @@ case "$COMMAND" in
         BUILD_DIR=$(mktemp -d -p /var/tmp)
         (
             [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && set -x
-            cd "$BUILD_DIR"
 
-            mkosi --default /usr/lib/mkosi-initrd/fedora.mkosi \
+            mkosi --include=/usr/lib/mkosi-initrd/mkosi.conf             \
```

Comment:
You can just specify `/usr/lib/mkosi-initrd` here and it will pick up the finalize script automatically.

Reply:
I think it's better to be explicit.